controllers/slice/slicerouter.go

```diff
@@ -262,7 +250,7 @@ func (r *SliceReconciler) getVolumeSpecForSliceRouter(s *kubeslicev1beta1.Slice,
 }
 
 // Creates a deployment spec for the vL3 slice router
-func (r *SliceReconciler) deploymentForSliceRouter(s *kubeslicev1beta1.Slice, ipamOctet string, dataplane string) *appsv1.Deployment {
```

Comment:
I think before these funcs are called we need to check if the cluster subnet is set. Please add a condition for clusterSubnetCidr in this func:
sliceConfigDefined()

The updates from the controller cluster could be staggered so there is no guarantee that the field would be set in the first few updates.

Reply:
https://github.com/kubeslice/worker-operator/pull/65/commits/55bf34190a07d863f0916328881aeb2d709609af this enough @bharath-avesha ?